### ISSUE TITLE
[no-var-requires] Recommendations do not show ES6 Modules example

### ISSUE THREAD

**Issue by ryanwilsonperkin** [typescript-eslint/typescript-eslint#899]
**Repro**

```JSON
{
  "rules": {
    "@typescript-eslint/no-var-requires": ["error"]
  }
}
```

```TS
const Image = require("./image.svg");
```

**Expected Result**
Documentation includes an example about ES6 Modules. ie.

```typescript
import Image from './image.svg';
```

**Actual Result**
Documentation only shows examples for typescript `import foo =` and `require(...)`

**Additional Info**
The [documentation for `no-var-requires`](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-var-requires.md) explains that `require` code can be replaced by ES6 modules `import` syntax, but does not give an example of it.

We've recently added support for importing from SVG files using ES6 module import syntax (using a webpack loader), but some developers still write using the `require` syntax. When they do, they get the lint error which then instructs them to write in a different syntax than we support. While the docs _do_ list ES6 modules as an option, they don't provide an example, which has confused some of our developers.

What I would love to see would be:

> **Examples of correct code for this rule:**
> ```diff
> import foo = require('foo');
> require('foo');
> + import foo from 'foo';
> ```

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `1.10.2` |
| `@typescript-eslint/parser`        | `1.10.2` |
| `TypeScript`                       | `3.2.2` |
| `ESLint`                           | `5.16.0` |
| `node`                             | `10.15.3` |
| `npm`                              | `6.4.1` |
